Code/Tools/ProjectManager/Source/GemCatalog/GemCatalogScreen.cpp

```diff
@@ -72,6 +74,16 @@ namespace O3DE::ProjectManager
         connect(m_headerWidget, &GemCatalogHeaderWidget::UpdateGemCart, this, &GemCatalogScreen::UpdateAndShowGemCart);
         connect(m_downloadController, &DownloadController::Done, this, &GemCatalogScreen::OnGemDownloadResult);
 
+        ScreensCtrl* screensControl = dynamic_cast<ScreensCtrl*>(parent);
+        if (screensControl)
+        {
+            ScreenWidget* createGemScreen = screensControl->FindScreen(ProjectManagerScreen::CreateGem);
+            if (createGemScreen)
+            {
+                connect(reinterpret_cast<CreateGem*>(createGemScreen), &CreateGem::GemCreated, this, &GemCatalogScreen::HandleGemCreated);
```

Comment:
This should be a `static_cast`.
We shouldn't be reinterpreting unrelated types to each other.

Reply:
is the `reinterpret_cast` need?